src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,62 @@
+.nav {
+  &__list {
+    display: flex;
+    justify-content: space-between;
+    padding: 0;
+  }
+
+  &__item {
+    list-style: none;
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+  }
+
+  &__link {
+    display: inline-block;
+    padding: 22.5px 0;
+
+    text-decoration: none;
+    text-transform: uppercase;
+    color: $main-font-color;
+    line-height: 15px;
+
+    transition: color 0.5s ease-out;
+
+    &:hover {
+      color: $blue-color;
+    }
+    //    is-active modifier according BEM
```

Comment:
You can remove all your comments. It's not necessary to leave them.

Reply:
ok